.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.7", "3.0", "3.1", "3.2"]
+        ruby: ["2.7", "3.0", "3.1", "3.2", "jruby", "truffleruby"]
```

Comment:
I recommend testing against their `head` versions instead so we can raise the issues we found to them, or receive addressed patches in time.